Automattic/wp-calypso
# Typography: Update instances of 10px, 11px, 12px to use $font-body-extra-small

#### Changes proposed in this Pull Request

* Update instances of 10px, 11px, and 12px fonts to use `$font-body-extra-small` which works out to 12px.
* There are a few instances of 10px and 11px that will get slightly larger with this change. So far, all instances I've found are an improvement for readability!

Some examples, not exhaustive:

| Before | After |
| ------- |------ |
| <img width="669" alt="before" src="https://user-images.githubusercontent.com/2124984/84698188-5e589f00-af1d-11ea-9d00-104e65b66928.png"> | <img width="667" alt="Screen Shot 2020-06-15 at 2 41 48 PM" src="https://user-images.githubusercontent.com/2124984/84698201-63b5e980-af1d-11ea-9676-2721a87c5d42.png"> |
| <img width="1059" alt="before2" src="https://user-images.githubusercontent.com/2124984/84698210-687a9d80-af1d-11ea-8161-57191c4d649f.png"> | <img width="1060" alt="Screen Shot 2020-06-15 at 3 06 19 PM" src="https://user-images.githubusercontent.com/2124984/84698220-6d3f5180-af1d-11ea-8235-8262f393e3b8.png"> |
| <img width="1680" alt="before3" src="https://user-images.githubusercontent.com/2124984/84698226-729c9c00-af1d-11ea-9f0e-41791463e4a9.png"> | <img width="1680" alt="Screen Shot 2020-06-15 at 2 31 48 PM" src="https://user-images.githubusercontent.com/2124984/84698235-78927d00-af1d-11ea-86d8-b4097230e6f5.png"> |
| <img width="396" alt="before4" src="https://user-images.githubusercontent.com/2124984/84698263-81834e80-af1d-11ea-9f5c-a0c9676e47e4.png"> | <img width="397" alt="Screen Shot 2020-06-15 at 3 23 38 PM" src="https://user-images.githubusercontent.com/2124984/84698274-86e09900-af1d-11ea-80d3-554896c4ac24.png"> |
| <img width="350" alt="before5" src="https://user-images.githubusercontent.com/2124984/84698322-9bbd2c80-af1d-11ea-84f5-e9f0eecd6ac5.png"> | <img width="343" alt="Screen Shot 2020-06-15 at 2 57 06 PM" src="https://user-images.githubusercontent.com/2124984/84698329-9f50b380-af1d-11ea-8090-24a813dc73e5.png"> |
| <img width="533" alt="before6" src="https://user-images.githubusercontent.com/2124984/84698341-a5469480-af1d-11ea-81f4-0ddeab5d4287.png"> | <img width="817" alt="Screen Shot 2020-06-15 at 3 14 37 PM" src="https://user-images.githubusercontent.com/2124984/84698350-a972b200-af1d-11ea-8362-879324ee6478.png"> | 
| <img width="380" alt="before7" src="https://user-images.githubusercontent.com/2124984/84698363-aecffc80-af1d-11ea-9dc8-ad18dd4aefb5.png"> | <img width="389" alt="Screen Shot 2020-06-15 at 3 24 43 PM" src="https://user-images.githubusercontent.com/2124984/84698373-b394b080-af1d-11ea-86ea-c24646126d56.png"> |
| <img width="1040" alt="before8" src="https://user-images.githubusercontent.com/2124984/84698383-b8f1fb00-af1d-11ea-980f-63f775c0adc7.png"> | <img width="1043" alt="Screen Shot 2020-06-15 at 3 25 49 PM" src="https://user-images.githubusercontent.com/2124984/84698393-bd1e1880-af1d-11ea-8bc3-9868decdf523.png"> |

#### Testing instructions

* Switch to this PR
* Check out screens across Calypso to look for instances of extra small fonts. Look for visual bugs or regressions.
